app/src/main/java/com/github/siela1915/bootcamp/Recipes/Utensils.java

```diff
@@ -43,6 +43,15 @@ public List<String> getUtensils() {
         return utensils;
     }
 
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj instanceof Utensils) {
+            return utensils.equals(((Utensils) obj).utensils);
```

Comment:
As you mentioned in UtensilsTest.java, we should compare the elements in the Utensils object, i.e. List<String>, regardless of the order. e.g. ['a', 'b', 'c'] is considered equal to ['b', 'a', 'c'] as they have the same elements though in different order.

You may consider change the logic to
```
if ((obj instanceof Utensils) && (((Utensils) obj).utensils.size() == utensils.size())) {
  for (String utensil : ((Utensils) obj).utensils) {
    if (!utensils.contains(utensil)) {
      return false;
    }
  }
  return true;
}
```